magicctapipe/scripts/lst1_magic/semi_automatic_scripts/database_production/LSTnsb.py

```diff
@@ -0,0 +1,180 @@
+"""
+Evaluates NSB level for a LST run
+"""
+import argparse
+import glob
+import logging
+import os
+
+import numpy as np
+import yaml
+from lstchain.image.modifier import calculate_noise_parameters
+
+__all__ = ["nsb"]
+
+logger = logging.getLogger(__name__)
+logger.addHandler(logging.StreamHandler())
+logger.setLevel(logging.INFO)
+
+
+def nsb(run_list, simtel, lst_config, run_number):
+
+    """
+    Here we compute the NSB value for a run based on a subset of subruns.
+
+    Parameters
+    ----------
+    run_list : list
+        List of subruns in the run
+    simtel : str
+        Simtel (MC) file to be used to evaluate the extra noise in dim pixels
+    lst_config : str
+        LST configuration file (cf. lstchain)
+    run_number : int
+        LST run number
+
+    Returns
+    -------
+    list
+        List of the sub-run wise NSB values
+    """
+
+    noise = []
+    denominator = 25
+    if len(run_list) == 0:
+        logger.warning(
+            "There is no subrun matching the provided run number. Check the list of the LST runs (LST_runs.txt)"
+        )
+        return
+    if len(run_list) < denominator:
+        mod = 1
+    else:
+        mod = int(len(run_list) / denominator)

```

Comment:
instead of casting to int you can use directly integer division: 
```mod = len(run_list) // denominator```

Reply:
Thanks (C++ casting habit vs. Python :facepalm: )

Reply:
Fixed